middleware_jwt_test.go

```diff
@@ -179,10 +179,10 @@ var jwtWithCentralDefNoPolicyBaseField = `
 
 `
 
-const JWTSECRET = "9879879878787878"
+const jwtSecret = "9879879878787878"
 
 // openssl genrsa -out app.rsa
-const JWTRSA_PRIVKEY = `
+const jwtRSAPrivKey = `
```

Comment:
I've never seen a style for constants in Go, and uppercase/underscores are often seen as bad style. Not a big fan of prefixes either, as it will make the names longer.

Why do we need to know whether something is a constant, exactly? Knowing whether a name is exported is useful, but in general I don't care whether something is a var or a const. If I need to modify it, it's for a particular reason and the compiler will yell at me anyway. Knowing it was a constant beforehand wouldn't have improved things.